src/ui/public/vislib/__tests__/visualizations/tile_maps/map.js

```diff
@@ -16,7 +16,7 @@ import VislibVisualizationsMapProvider from 'ui/vislib/visualizations/_map';
 //   ['rows', require('fixtures/vislib/mock_data/geohash/_rows')],
 // ];
 
-// // TODO: Test the specific behavior of each these
+//
```

Comment:
i added the comment back. there were generally no changes done to tile maps (except minor things to make it work with the rest of refactor). With Thomas we intend to move maps out of vislib in near future and do some refactoring on it as well. At that time we'll also try to add more extensive tests.